.github/workflows/build.yaml

```diff
@@ -62,7 +62,7 @@ jobs:
       ROX_PRODUCT_BRANDING: ${{ matrix.branding }}
     runs-on: ubuntu-latest
     container:
-      image: quay.io/stackrox-io/apollo-ci:stackrox-test-0.3.58
+      image: quay.io/stackrox-io/apollo-ci:stackrox-test-0.3.59
```

Comment:
How about extracting `0.3.59` into variable so next time we need to change it in one place